arches_lingo/pkg/graphs/resource_models/Concept.json

```diff
@@ -1985,31 +2133,6 @@
                     "visible": true,
                     "widget_id": "ff3c400a-76ec-11e7-a793-784f435179ea"
                 },
-                {
-                    "card_id": "bf73e630-4888-11ee-8a8d-11afefc4bff7",
```

Comment:
Is it expected that this Card is now deleted, also?

Reply:
No, in the example I posed [here](https://github.com/archesproject/arches/pull/12063#pullrequestreview-2861571598), this is the `Warrant Assertion Event` card, which has other data collecting nodes.